assets/js/pages/ClusterDetails/ClustersList.test.jsx

```diff
@@ -8,25 +8,58 @@ import { renderWithRouter, withState } from '@lib/test-utils';
 
 import ClustersList from './ClustersList';
 
+const cleanInitialState = {
+  hostsList: {
+    hosts: [],
+  },
+  clustersList: {
+    clusters: [],
+  },
+  sapSystemsList: {
+    sapSystems: [],
+    applicationInstances: [],
+    databaseInstances: [],
+  },
+  databasesList: {
+    databaseInstances: [],
+  },
+  user: {
+    abilities: [{ name: 'all', resource: 'all' }],
+  },
+};
+
 describe('ClustersList component', () => {
-  describe('filtering', () => {
-    const cleanInitialState = {
-      hostsList: {
-        hosts: [],
-      },
-      clustersList: {
-        clusters: [],
-      },
-      sapSystemsList: {
-        sapSystems: [],
-        applicationInstances: [],
-        databaseInstances: [],
-      },
-      databasesList: {
-        databaseInstances: [],
-      },
-    };
+  describe('tags operations', () => {
+    it('should disable tag creation and deletion if the user abilities are not compatible', async () => {
+      const state = {
+        ...cleanInitialState,
+        clustersList: {
+          clusters: [].concat(
```

Comment:
Just remove the concat. You could directly assign the `buildList` output

Reply:
done